### PR TITLE
Fix Issue 18427 - Improve deprecation message for importing privately imported symbol

### PR DESCRIPTION
**Pull request by RazvanN7**
The compiler cannot show how to import it because there is no way to import a symbol which is privately declared or imported in another module. The only way is to make the symbol/import public, but the compiler cannot advise that, so I pointed out where the variable is declared/imported.

### CODE REVIEW

File: src/dmd/dsymbolsem.d
Comment (JinShil) [inline src/dmd/dsymbolsem.d]: This comment needs to be updated to "following 5 lines", or wordsmithed in a way to where the number of lines doesn't matter.

**Review by JinShil — APPROVED** [review]
LGTM other than the comment nit.

**Review by wilzbach — APPROVED** [review]
LGTM too (module @JinShil's nit)

File: src/dmd/dsymbolsem.d
Comment (RazvanN7) [reply src/dmd/dsymbolsem.d]: Oh, missed that, sorry. Thanks for the awesome attention to details. I addressed it now.

**Comment by RazvanN7** [comment]
ping @wilzbach @JinShil I think this is good to go now. Merging this will make the PR queue < 140  :dagger:

**Comment by JinShil** [comment]
It's relatively new PR, so I just want to give it at least 24 hours before I put my name next to the merge button.  If someone else wants to cowboy it in right now, I won't object, but I want to make sure there's sufficient time for others to have their say.  Also Jenkins is failing for some reason.  I'll restart it.

**Pull request merged**
